core/src/test/java/org/apache/lucene/queries/BlendedTermQueryTests.java

```diff
@@ -205,7 +206,7 @@ public void testBasics() {
     public Term[] toTerms(String[] fields, String term) {
         Term[] terms = new Term[fields.length];
         List<String> fieldsList = Arrays.asList(fields);
-        Collections.shuffle(fieldsList, random());
+        Randomness.shuffle(fieldsList);
```

Comment:
I don't understand why the new method would ever need to be used in tests. random() can safely be used and is always more clear.

We should only use the Randomness method for non-test code, and keep it contained. Collections.shuffle(List, Random) is always ok.